apps/next-evans/components/chrome/algolia/DropdownSearchResults.tsx

```diff
@@ -61,6 +63,12 @@ const DropdownSearchResults: React.FC<Props> = ({
           <Pagination />
         </Index>
       </div>
+      <div className="flex px-2 pb-2 justify-end items-center gap-1">
+        <span className="text-sm text-gray-400">
+          {LANG === `en` ? `Search by Algolia` : `Buscar por Algolia`}
+        </span>
+        <Image src={AlgoliaLogo} alt="Algolia logo" className="w-8 h-8" />
```

Comment:
i looked over the tos and branding doc, and i think they're ok with us doing a monochrome version of the logo, the language isn't super strong that we have to use their blue.  so, can we change this to black? maybe even just by adding `filter: grayscale(1)`, i don't love the blue color, it kind of clashes with the rest